src/index.html

```diff
@@ -10,13 +10,101 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <img
+        src="images/logo.png"
+        alt="Moyo logo"
+        class="logo"
+      />
+      <nav>
+        <ul class="nav__links__list">
+          <li>
+            <a
+              href="index.html"
+              class="link-item, is-active"
```

Comment:
There is a comma in the class attribute value 'link-item, is-active'. Class names should be separated by spaces, not commas. Please change it to 'link-item is-active'.